ci/do-audit.sh

```diff
@@ -33,6 +33,11 @@ cargo_audit_ignores=(
 
   # mio
   --ignore RUSTSEC-2024-0019
+
+  # curve25519-dalek
+  # Patches to address the advisory have been pulled into a fork of the repo.
+  # See Cargo.toml for more information
```

Comment:
```suggestion
  # See `[patch.crates-io.curve25519-dalek]` in Cargo.toml for more information
```

Reply:
Done - made the change myself to make sure you aren't an author on the PR and your review will count for pushing this